pkg/server/plugin/nodeattestor/k8ssat/sat.go

```diff
@@ -262,6 +262,8 @@ func (p *AttestorPlugin) getNamesFromClaims(claims *k8s.SATClaims) (namespace st
 }
 
 func (p *AttestorPlugin) Configure(_ context.Context, req *configv1.ConfigureRequest) (*configv1.ConfigureResponse, error) {
+	p.log.Warn(fmt.Sprintf("The %q node attestor plugin has been deprecated in favor of the \"k8s_psat\" plugin and will be removed in a future release", pluginName))
```

Comment:
```suggestion
	p.log.Warnf("The %q node attestor plugin has been deprecated in favor of the \"k8s_psat\" plugin and will be removed in a future release", pluginName)
```
nit

Reply:
Forgot this was the plugin logger using the hclog interface, so we can't do what I'm proposing here.